### PR TITLE
Tandem updates

### PR DESCRIPTION
**Pull request by gniezen**
See attached Trello card for details.

### CODE REVIEW

**Comment by gniezen** [comment]
[Tandem Predictive Low Glucose Suspend](https://trello.com/c/fNigcnTf/462-tandem-predictive-low-glucose-suspend)

File: app/reducers/devices.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const devices = {
     key: 'tandem',
     name: 'Tandem',
     source: {type: 'device', driverId: 'Tandem'},
-    enabled: {mac: true, win: true}
+    enabled: {mac: true, win: true, linux: true}
```
Comment (pazaan) [inline app/reducers/devices.js]: Linux support is always welcome 😄

File: lib/drivers/tandem/tandemDriver.js [hunk 2]
```diff
@@ -1934,7 +1949,13 @@ module.exports = function (config) {
   };
 
   var buildCGMRecords = function(data, records) {
-    var cgmRecords = filterLogEntries([PUMP_LOG_RECORDS.LID_CGM_DATA,PUMP_LOG_RECORDS.LID_CGM_CALIBRATION,PUMP_LOG_RECORDS.LID_CGM_DATA_GX,PUMP_LOG_RECORDS.LID_CGM_CALIBRATION_GX], data.log_records);
+    var cgmRecords = filterLogEntries([
```
Comment (pazaan) [inline lib/drivers/tandem/tandemDriver.js]: Nice readability update 👌

**Comment by gniezen** [comment]
@pazaan OK, I promise I'll try not to make any changes to this PR :stuck_out_tongue_closed_eyes: Please review changes since last approval.

**Comment by pazaan** [comment]
@gniezen Sure, sure 😜

**Pull request merged**